src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/DocumentHelperTest.java

```diff
@@ -1363,6 +1367,83 @@ public void buildDocumentContent15() {
                 null, null).toString());
     }
 
+    @Test
+    public void buildDocumentContent20() {
```

Comment:
expected strings are so long... May be write them in a json?

Reply:
I have a look.

Reply:
Done